src/metrics-collector/db/sqldb/sqldb.go

```diff
@@ -0,0 +1,153 @@
+package sqldb
+
+import (
+	"database/sql"
+	"encoding/json"
+	"strconv"
+
+	_ "github.com/lib/pq"
+	"github.com/pivotal-golang/lager"
+
+	"metrics-collector/config"
+	"metrics-collector/metrics"
+)
+
+const PostgresDriverName = "postgres"
+
+type SQLDB struct {
+	database *sql.DB
+	logger   lager.Logger
+	conf     *config.DbConfig
+}
+
+func NewSQLDB(conf *config.DbConfig, logger lager.Logger) (*SQLDB, error) {
+	sqldb := &SQLDB{}
+	sqldb.conf = conf
+	sqldb.logger = logger
+
+	var err error
+	sqldb.database, err = sql.Open(PostgresDriverName, conf.Url)
+	if err != nil {
+		logger.Error("Connect-to-database", err)
+		return nil, err
+	}
+
+	err = sqldb.initMetricsTable()
+	if err != nil {
+		logger.Error("initialize-metrics-table", err)
+		return nil, err
+	}
+
+	return sqldb, err
+}
+
+func (db *SQLDB) initMetricsTable() error {
```

Comment:
We aren't doing this in code but via liquidb.
You should have a db/metrics.db.changelog.yml which describes your table schema.